### PR TITLE
Create Lightning v0.30.0 RC branch

### PR DESCRIPTION
**Pull request by mlxd**
### Before submitting

Please complete the following checklist when submitting a PR:

- [ ] All new features must include a unit test.
      If you've fixed a bug or added code that should be tested, add a test to the
      [`tests`](../tests) directory!

- [ ] All new functions and code must be clearly commented and documented.
      If you do make documentation changes, make sure that the docs build and
      render correctly by running `make docs`.

- [ ] Ensure that the test suite passes, by running `make test`.

- [ ] Add a new entry to the `.github/CHANGELOG.md` file, summarizing the
      change, and including a link back to the PR.

- [ ] Ensure that code is properly formatted by running `make format`. 

When all the above are checked, delete everything above the dashed
line and fill in the pull request template.

------------------------------------------------------------------------------------------------------------

**Context:**

**Description of the Change:**

**Benefits:**

**Possible Drawbacks:**

**Related GitHub Issues:**

### CODE REVIEW

File: pennylane_lightning/_version.py [hunk 1]
```diff
@@ -17,4 +17,4 @@
 """
 
 
-__version__ = "0.30.0-dev7"
+__version__ = "0.30.0rc0"
```
Comment (AmintorDusko) [inline pennylane_lightning/_version.py:20]: Smal suggestion:
```suggestion
__version__ = "0.30.0-rc0"
```

File: pennylane_lightning/_version.py [hunk 2]
```diff
@@ -17,4 +17,4 @@
 """
 
 
-__version__ = "0.30.0rc0"
+__version__ = "0.30.0-rc0"
```
Comment (mlxd) [inline pennylane_lightning/_version.py]:
```suggestion
__version__ = "0.30.0-rc1"
```

Comment (mlxd) [reply pennylane_lightning/_version.py]: We need to update this to rc1 now that the changes are into master.

Comment (AmintorDusko) [reply pennylane_lightning/_version.py]: Sure makes sense. I will commit this suggestion.

**Review by vincentmr — APPROVED** [review]
Looks good to me. Cheers.

**Review by multiphaseCFD — APPROVED** [review]
Looks good to me! Thanks @mlxd !

**Comment by mlxd** [comment]
Forked as `v0.30.0_release` and released with tag `v0.30.0`. All changes to merge will master will be included in PR #446. Closing.

**Pull request closed without merging**